### PR TITLE
Task/forma validacion login

### PR DESCRIPTION
**Pull request by AndreeToledo**
#### Where should the reviewer start?
It creates the views, and methods necessary to login a user.

#### How should this be manually tested?
The tester should create a user, and try to login at localhost:8000/login, then test invalid credentials, and then test submitting the form with empty fields.

#### Any background context you want to provide?
N/A

This template was adapted ~stolen~ from [Quickleft/Sprint.ly](https://quickleft.com/blog/pull-request-templates-make-code-review-easier/)

### CODE REVIEW

File: jp2_online/templates/base/home.html [hunk 1]
```diff
@@ -15,15 +15,15 @@
 
 <div class='row text-center'>
 	{% if user and not user.is_anonymous %}
-	<a>Hello, {{ user.get_full_name }}!</a>
+	<a><h4>Hello, {{ user.get_full_name }}!</h4></a>
 	<br>
 	<a href="/tosp_auth/logout">Logout</a>
 	{% else %}
-	Social login used to be here.
+	<a href="/tosp_auth/login">Inciar Sesión</a>
```
Comment (erikiado) [inline jp2_online/templates/base/home.html]: typo: Iniciar Sesion.
Esta vista se mantendra?

File: jp2_online/templates/tosp_auth/login_form.html [hunk 2]
```diff
@@ -0,0 +1,28 @@
+<div class="row text-center">
```
Comment (erikiado) [inline jp2_online/templates/tosp_auth/login_form.html]: Can this file be implemented in the style of the template? here is the download link: https://codeload.github.com/VinceG/Bootstrap-Admin-Theme-3/zip/master

File: jp2_online/templates/base/home.html (hunk 1 above)
Comment (AndreeToledo) [inline jp2_online/templates/base/home.html]: Sure, I think this view is very useful, we can put the login form in a modal, but if they introduce wrong credentials, the easiest thing to do is to redirect them to this view, and also simplifies the presentation of error messages.

I'll fix the typo

File: jp2_online/templates/tosp_auth/login_form.html (hunk 2 above)
Comment (AndreeToledo) [inline jp2_online/templates/tosp_auth/login_form.html]: Do you mean like this?

http://vinceg.github.io/Bootstrap-Admin-Theme-3/login.html

Comment (erikiado) [inline jp2_online/templates/tosp_auth/login_form.html]: yes, so we don't need to worry later on about fixing front end

**Pull request merged**